### PR TITLE
Move logged out components into a wrapper

### PR DESCRIPTION
**Pull request by adenix**

### CODE REVIEW

File: angular-cli.json [hunk 1]
```diff
@@ -15,11 +15,11 @@
       "prefix": "app",
       "mobile": false,
       "styles": [
-        "assets/styles/style.scss",
-        "assets/styles/spinner.scss",
         "../node_modules/@angular2-material/core/overlay/overlay.css",
         "../node_modules/bootstrap/dist/css/bootstrap.css",
-        "../node_modules/font-awesome/css/font-awesome.css"
+        "../node_modules/font-awesome/css/font-awesome.css",
+        "assets/styles/style.scss",
```
Comment (iveysaur) [inline angular-cli.json]: why though

File: angular-cli.json [hunk 2]
```diff
@@ -28,7 +28,8 @@
       "environments": {
         "source": "environments/environment.ts",
         "dev": "environments/environment.ts",
-        "prod": "environments/environment.prod.ts"
+        "prod": "environments/environment.prod.ts",
```
Comment (iveysaur) [inline angular-cli.json]: also why is this a change in yours? i thought this was a merged change?

Comment (adenix) [inline angular-cli.json]: I'm not sure why this showed up as my change. Git is weird.

File: angular-cli.json (hunk 1 above)
Comment (adenix) [inline angular-cli.json]: Because material icons don't have the facebook or google + icon. I'm sure we will need it somewhere else.

**Comment by adenix** [comment]
Updated breakpoints to fix image cut off

File: angular-cli.json (hunk 2 above)
Comment (adenix) [inline angular-cli.json]: Looks like you committed on this branch somehow.

File: angular-cli.json (hunk 1 above)
Comment (iveysaur) [inline angular-cli.json]: sorry I meant why move it to the bottom

File: angular-cli.json (hunk 2 above)
Comment (iveysaur) [inline angular-cli.json]: Yeaa i definitely didn't, git is just being weird

**Pull request merged**